app/src/features/reports/CompareStatesForVariableReport.tsx

```diff
@@ -10,18 +10,19 @@ import { Breakdowns } from "../../utils/Breakdowns";
 import VariableProvider from "../../utils/variables/VariableProvider";
 
 function CompareStatesForVariableReport(props: {
-  state1: string;
-  state2: string;
+  stateFips1: string;
+  stateFips2: string;
```

Comment:
Some musings, that are not blocking this PR but I think would be nice to do at some point. I'm thinking we should transform all geographic data returned from `VariableProvider.getData()` into a fips_code column and a geo_name column, so that we can smoothly compare any geography level to any other geography level. Right now it feels wrong to be adding the US as a "state", and that will only get weirder when we add counties.

Reply:
Yeah that sounds good to me!